packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -200,8 +200,8 @@ export function makeScreenshotEditorComponent({ imageBuffer, dialog, options }:
           (croppingBox.height / imageBox.height) * imageBuffer.height,
           0,
           0,
-          croppingBox.width,
-          croppingBox.height,
+          croppingBox.width * DPI,
+          croppingBox.height * DPI,
```

Comment:
Should we make this...

```suggestion
          cutoutCanvas.width,
          cutoutCanvas.height,
```

Or save the calc to an intermediate var